src/main/java/org/gbif/api/model/registry/Dataset.java

```diff
@@ -264,6 +269,45 @@ public void setPublishingOrganizationKey(UUID publishingOrganizationKey) {
   }
 
   /**
+   * Persisted in the database table.
+   *
+   * @return the frequency with which changes are made to the dataset
+   */
+  @Nullable
+  public MaintenanceUpdateFrequency getMaintenanceUpdateFrequency() {
+    return maintenanceUpdateFrequency;
+  }
+  /**
+   * Persisted in the database table.
+   */
+  public void setMaintenanceUpdateFrequency(MaintenanceUpdateFrequency maintenanceUpdateFrequency) {
+    this.maintenanceUpdateFrequency = maintenanceUpdateFrequency;
+  }
+
+  /**
+   * TODO add @NotNull when all datasets have a license applied to them
```

Comment:
Thanks. Upon your suggestion I created [POR-3133](http://dev.gbif.org/issues/browse/POR-3133). This Jira also scopes out the full set of registry changes needed to ensure all datasets get assigned a license, the registry web services are in good working order, etc.